packages/search-types/src/redirection.model.ts

```diff
@@ -9,7 +10,7 @@ import { Tagging } from './tagging.model';
  *
  * @public
  */
-export interface Redirection extends Identifiable {
+export interface Redirection extends NamedModel<'Redirection'>, Identifiable {
```

Comment:
We should for code consistencies shake. I created the task [here](https://searchbroker.atlassian.net/browse/EX-4733)

Reply:
There you are using `Object.assign`, which allows you to assign anything, it does not have to match the first parameter type. We used it in the past to avoid casting types to `any`.

![image](https://user-images.githubusercontent.com/68222542/133255947-4062fcdf-26af-4313-80d6-dbb2090a68fe.png)

Moreover the `modelName` property is read only.

![image](https://user-images.githubusercontent.com/68222542/133256108-b732767a-8750-4c98-a565-8e5fb2adf999.png)

Reply:
I tried with the `as Redirection` in the object assign but the result was the same, thanks.